systests/cluster/test_remote_account.py

```diff
@@ -579,9 +579,10 @@ def test_monitor_log_exception(self):
     @cluster(num_nodes=1)
     def test_kill_process(self):
         """Tests that kill_process correctly works"""
+        grep_str = '"nc -l -p 5000"'
 
         def get_pids():
-            pid_cmd = "ps ax | grep -i nc | grep -v grep | awk '{print $1}'"
+            pid_cmd = f"ps ax | grep -i {grep_str} | grep -v grep | awk '{{print $1}}'"
```

Comment:
nit: should we quote the grep string here for clairity up to you will approve regradless?